### DISCUSSION TITLE
Audio from browser to server

### DISCUSSION THREAD

**Discussion by z3nb4k1** [bluenviron/mediamtx#1512] [Questions]
Thank you very much for the excellent work and also for keeping it so updated !!
With FFMPEG I have managed to publish audio stream on the remote web server (RTSP - HTTPS). Fantastic !
How could I achieve something similar, but publishing audio stream captured via web browser and microphone ?

**Comment by aler9** [comment]
Hello, this may be a future enhancement, i've created a feature request in order to track the development: #1659
